travis-setup.md

```diff
@@ -0,0 +1,96 @@
+# Building and Deploying with Travis
+
+### Setup Travis web
+ 1) To get started with Travis, follow the first 3 steps in the following tutorial:
+    - https://docs.travis-ci.com/user/tutorial/#to-get-started-with-travis-ci
+    - For step 3, make sure you include activating your fork of this repository
+
+### Create Firebase account
+ 1) Navigate to https://console.firebase.google.com
+ 2) Sign in/make an account
+ 3) Create a new project:
+    - For project name, use `{your username}-tba-dev`
+    - Leave the other settings to the defaults, unless you know what you're doing
+
+    ![](https://i.imgur.com/tk4kWtn.png)
+
+ 4) On the left, navigate to Develop -> Storage
+    - After clicking through a few confirmations, you should be on a screen like this:
+
+    ![](https://i.imgur.com/tpFcxrU.png)
+
+5) Copy your storage URL for later use:
+   - Copy the storage URL, which is `gs://dracco1993-tba-dev-17792.appspot.com` in this example, to some place you'll remember; we'll need this URL in a later step.
+
+### Create/Download Firebase keys
+ 1) On the top left of Firebase, click the gear icon, then go to Project Settings
+ 2) From here, go to the Service Accounts tab
+ 3) Click "Generate new private key", then confirm by clicking "Generate key"
+    - Generating a key in this manner will download a new `.json` file, containing all the information required to access this Firebase project via an API
+    - As the warning in Firebase says, "Keep it confidential and **never store it in a public repository**"
+
+### Encrypt Firebase keys with Travis CLI
+ 1) Setup the Travis CLI
+    - https://github.com/travis-ci/travis.rb#installation
+ 2) Inside this repository's directory:
```

Comment:
Can you explain why this step is necessary?

Reply:
(in the docs - that is)

Reply:
I've added additional explanation here: 3728cbf